frontend/arrangor-flate/turbo.json

```diff
@@ -1,7 +1,27 @@
 {
   "extends": ["//"],
   "tasks": {
+    "api-client:build": {
+      "outputs": ["api-client"]
+    },
+    "backend": {
+      "dependsOn": ["api-client:build"],
+      "outputs": []
+    },
+    "dev": {
+      "dependsOn": ["api-client:build"],
+      "outputs": []
+    },
+    "lint:fix": {
+      "dependsOn": ["api-client:build"],
+      "outputs": []
+    },
+    "typecheck": {
+      "dependsOn": ["api-client:build"],
+      "outputs": []
+    },
```

Comment:
Usikker på om man trenger alle disse, skjønner ikke helt hvorfor turbo dev i mr-admin-flate bygger api-client-v2? Fordi den er i dependencies? For uten dette blir det ikke bygd først her

Reply:
api-client-v2 blir bygd pga avhengighet til `^build` å root turbo.json:

```
    "typecheck": {
      "dependsOn": ["^build"],
      "outputs": []
    },
```

Reply:
Men jeg står i mr-admin-flate og kjører typecheck? mulighetsrommet-veileder-flate blir jo ikke bygget?

Reply:
eller arrangor-flate. Den har også build, hvordan vet den at bare api-client-v2 skal bygges